satellite-clone-vars.sample.yml

```diff
@@ -17,9 +17,9 @@
 # Disable firewall to setup Satellite for testing purposes. (defaults to false)
 #disable_firewall: false
 
-# Run katello reindex after the Satellite install. (defaults to false)
+# Run `foreman-rake katello:reimport` after the Satellite install. (defaults to false)
 # Note: If you are planning to clone your Satellite and immediately upgrade, you may choose to skip this step.
-#run_katello_reindex: false
+#run_katello_reimport: false
```

Comment:
Users can have this config file in `/etc`. Do we need to worry about those things? I really don't know what our support policy is in this repo.

Reply:
I think this is fine?

cc @mccun934